modules/ROOT/pages/deployment/general/roles-permissions.adoc

```diff
@@ -0,0 +1,128 @@
+
+Compared to ownCloud Server where e.g. the administrator has more or less full access to everything, Infinite Scale refines roles with assigned permissions types and particular permissions for that type on a much more detailed level. This is not only true for system roles abut also for spaces roles.
+
+=== Security
+
+With role and permission separation (system/spaces/sharing), the overall security is greatly increased and security standards like *separating tasks and data access by design* is addressed.
+
+== System Roles and Permissions
+
+Currently there are four predefined system roles with fixed permissions assigned to specific roles. Note that some permissions are only granted via the role for the particular user but not covering all users. A defined role can be assigned to a user, concatenating roles is currently not supported.
+
+Consider that these roles and permissions are provided by the system via APIs, which means you can call them via own code or `curl`. 
+
+Read the image below the following way::
+* A Role gets one or more permission types assigned.
+* Each permission type is granted individual rights relevant for that role.
+* Roles are assigned to users.
+
+// the empty macro is necessary to have more distance between text and image
+ 
+{empty} +
+
+// the table references following ocis code where roles and permissions a re defined: https://github.com/owncloud/ocis/blob/master/services/settings/pkg/store/defaults/defaults.go
+
+image::deployment/general/roles-and-permissions-matrix.svg[width=550]
+
+=== System Role Details
+
+In general with regards to spaces:
+
+* Spaces can always be deleted by the those who are granted managing the space with the exception of the `Delete Home Space` permission, which is only assigned to the Admin for the reason when a user account gets deleted.
+* Managing a space means creating or deleting a space, setting quotas, delegating the right to manage space members and space member roles, but not having access to the content of the space except when being a member of the space.
+* When creating a space, the creator is automatically in the delegatable Manager role for this space. For details see xref:space-roles-and-permissions[Space Roles and Permissions].
+
+Admin::
+The `Admin` role is in full control of all permission types with one exception which is `Self Management`. This permission type is excluded to avoid that the last admin accidentally reduces their own role to a lower one making the system unmanageable. As long as there is more than one admin, roles can be set by the `Role Management` permission.
+
+Space Admin::
+The `Space Admin` role allows you to create spaces, set quotas and delegating the right to manage space members and space member roles.
+
+User::
+The `User` role can manage his own account and change his language. To manage space group members and space member roles, an Admin or Space Admin must create one and grant the Manager permission to the user.
```

Comment:
```suggestion
If you have the `User` role assigned, you can manage your own account and change the language. To manage space group members and space member roles, an Admin or Space Admin must create one and grant the Manager permission to the user.
```

Reply:
Not sure what the last sentence is supposed to say. What needs to be created, a space or a user? Is there such a thing as space groups?